### ISSUE TITLE
Understanding yolo_loss usage.

### ISSUE THREAD

**Issue by gustavovaliati** [qqwweee/keras-yolo3#221]
@qqwweee congratulations for the great work you have done! :1st_place_medal: 

I am looking for some advice.

I am creating an extra head in the yolo architecture to implement and additional output of semantic segmentation. It is basically done.

https://github.com/gustavovaliati/keras-yolo3/blob/ff8d2701d9520f20dc060f0d434d79e083b2df40/yolo3/model.py#L88

However, I am struggling with the loss function. Seems like my segmentation head is not learning. I know the problem could be in many other places, but I have checked many of them and the loss is my current target.

I wonder whether I should sum up my segmentation loss value with the yolo loss, as I did here

https://github.com/gustavovaliati/keras-yolo3/blob/7b16a6c551e996727d46e4a03d4637bf44716c37/yolo3/model.py#L506

Or should I do something similar with the keras instructions (https://keras.io/getting-started/functional-api-guide/#multi-input-and-multi-output-models) where we would compile yolo_loss and segmentation loss apart acoording to the outputs? This one I have not been able to test due the way you have linked the yolo_loss with the model. I have read the keras source code, and I still do not understand how this following lambda works:

https://github.com/qqwweee/keras-yolo3/blob/e6598d13c703029b2686bc2eb8d5c09badf42992/train.py#L73

So there are two questions:
1. Should I sum my own segmentation loss value with the yolo_loss inside the default loss function?
2. How does the lambda works in the model.compile?

Thank you very much.
